Mathlib/RingTheory/Fintype.lean

```diff
@@ -12,6 +13,43 @@ import Mathlib.Data.Fintype.Units
 -/
 
 
+open Finset ZMod
+
+variable {R : Type*} [Ring R] [Fintype R] [DecidableEq R]
+
+lemma Finset.univ_of_card_eq_two (h : Fintype.card R = 2) :
+    (univ : Finset R) = {0, 1} := by
+  have : Nontrivial R := by
+    refine Fintype.one_lt_card_iff_nontrivial.1 ?_
+    rw [h]
+    norm_num
+  refine (eq_of_subset_of_card_le (subset_univ _) ?_).symm
+  rw [card_univ, h, card_insert_of_not_mem, card_singleton]
+  rw [mem_singleton]
+  exact zero_ne_one
+
+lemma Finset.univ_of_card_eq_three (h : Fintype.card R = 3) :
```

Comment:
Not sure if it will ever be useful... but I guess you can weaken the `= 3` to `\le 3`. Idem for the `= 2` above.

Reply:
Why not!